rust/agents/relayer/src/relayer.rs

```diff
@@ -88,7 +90,25 @@ impl BaseAgent for Relayer {
 
         let whitelist = parse_matching_list(&settings.whitelist);
         let blacklist = parse_matching_list(&settings.blacklist);
-        info!(whitelist = %whitelist, blacklist = %blacklist, "Whitelist configuration");
+
+        let skip_transaction_gas_limit_for = settings
+            .skiptransactiongaslimitfor
+            .map(|l| {
+                l.split(',')
+                    .map(|d| {
+                        d.parse()
+                            .expect("Error parsing domain id for transaction gas limit")
+                    })
+                    .collect()
+            })
+            .unwrap_or_default();
+
+        let transaction_gas_limit = settings
+            .transactiongaslimit
+            .map(|l| l.parse())
+            .transpose()
+            .context("Invalid transaction gas limit")?;
+        info!(%whitelist, %blacklist, ?transaction_gas_limit, "Whitelist configuration");
```

Comment:
nit: maybe worth adding `skip_transaction_gas_limit_for` here & updating the log msg